src/_includes/banner.html

```diff
@@ -1,8 +1,11 @@
-<div class="banner">
-    <p class="banner__text">
-	Join the Dart team:
-        <a target="_blank" 
-          href="https://www.google.com/about/careers/jobs#t=sq&q=j&li=20&l=false&j=dart&jcoid=7c8c6665-81cf-4e11-8fc9-ec1d6a69120c&jcoid=e43afd0d-d215-45db-a154-5386c9036525">Become
-	    a Developer Advocate</a>
-    </p>
+{% if site.url == 'https://www.dartlang.org' %}
+{% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
+<div class="banner alert alert-info">
+  <p class="banner__text">
+    Welcome to <b>Dart 2</b>!
+    Looking for Dart 1.x? See the site
```

Comment:
How about:

Welcome to **Dart 2**! Still using Dart 1.x? See the [archived site]() or the [migration guide]().

Reply:
It feels like Dart 2 should be a link to something. But what?

Reply:
Changes done and staging server refreshed.